games-util/springlobby/springlobby-0.264.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit cmake-utils eutils flag-o-matic wxwidgets
+
+DESCRIPTION="lobby client for spring rts engine"
+HOMEPAGE="http://springlobby.info"
+SRC_URI="http://www.springlobby.info/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
+IUSE="+translation debug +libnotify"
+
+RDEPEND="
+	>=dev-libs/boost-1.35
+	x11-libs/wxGTK:3.0
+	net-misc/curl
+	libnotify? (    x11-libs/libnotify )
+	translation? ( sys-devel/gettext )
+	media-libs/alure
+"
+
+DEPEND="${RDEPEND}
+	>=sys-devel/gcc-4.7
+	>=dev-util/cmake-2.8.11
```

Comment:
cmake-utils takes care of depending on CMake.

Reply:
Ok, removed